MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -25,9 +25,9 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
-    var leg: RouteLeg? {
+    weak var leg: RouteLeg? {
```

Comment:
Why does this property need to be weak? If `leg` is set using a local variable, it’ll unexpectedly become `nil` right after the local variable goes out of scope. (Same for `step`, for that matter.)